packages/hyperion-react-testapp/src/AutoLoggingWrapper.ts

```diff
@@ -95,6 +95,15 @@ export function init() {
           eventName: 'change',
           cacheElementReactInfo: true,
           interactableElementsOnly: false,
+        },
+        {
+          eventName: 'mouseover',
+          cacheElementReactInfo: true,
+          interactableElementsOnly: false,
```

Comment:
One option would be to add something like `durationThresholdToEmitHoverEvent` to the `mouseover` config itself. 
Then, user can enable mouseover and if they also want separate hover event, add this one flag. 
In the hover publisher, you can then read this (just like `change` for the element value) and decide to publish or not. 
That might make the config management easier. 

We still may need a "derived" or composite event config style for future. But for now, this simplification might work.